.github/workflows/ci.yml

```
@@ -49,4 +46,4 @@ jobs:
         with:
           sarif_file: rust-clippy-results.sarif
           wait-for-processing: true
-     
\ No newline at end of file
+     
```

yamllint

[error] trailing spaces

(trailing-spaces)